src/CSRMultiplication/SparseMatrixMultiplication.fs

```diff
@@ -0,0 +1,206 @@
+namespace CSRMultiplication
+
+open Brahma.OpenCL
+open Brahma.FSharp.OpenCL.Core
+open Brahma.FSharp.OpenCL.Extensions
+
+module SparseMatrixMultiplication = 
+    let multiplySpMV 
+        (provider: ComputeProvider) 
+        (commandQueue: CommandQueue) 
+        (vector: float[]) 
```

Comment:
Can these operations be generic? Yes, we can not define new semirings, but we can use all defined.